bin/kubectl-crossplane-package-init

```diff
@@ -111,8 +111,8 @@ spec:
       containers:
       - name: "${KUBEY_PACKAGE_NAME}-controller"
         # The `image:` field is optional. If omitted, it will be
-        # filled in by the stack manager, using the same image
-        # name and tag as the package on the StackInstall object.
+        # filled in by the package manager, using the same image
```

Comment:
@hasheddan @jbw976 if time permits, would be good to update other references to "stacks manager" -> "package manager", including a rev of the design doc.

Reply:
@prasek agreed, although it will be lower priority than any testing and functional changes :+1: